services/terrain/doc/endpoints/misc.md

```diff
@@ -17,23 +17,23 @@
     * [Setting a user's saved searches](#setting-saved-searches)
     * [Deleting a user's saved searches](#deleting-saved-searches)
 
-# Miscellaneous Donkey Endpoints
+# Miscellaneous Terrain Endpoints
 
-Note that secured endpoints in Donkey and metadactyl are a little different from
-each other. Please see [Donkey Vs. Metadactyl](donkey-v-metadactyl.md) for more
+Note that secured endpoints in Terrain and metadactyl are a little different from
+each other. Please see [Terrain Vs. Metadactyl](terrain-v-metadactyl.md) for more
 information.
 
-## Verifying that Donkey is Running
+## Verifying that Terrain is Running
 
 Unsecured Endpoint: GET /
 
-The root path in Donkey can be used to verify that Donkey is actually running
+The root path in Terrain can be used to verify that Terrain is actually running
 and is responding. Currently, the response to this URL contains only a welcome
 message. Here's an example:
 
 ```
 $ curl -s http://by-tor:8888/
-Welcome to Donkey!  I've mastered the stairs!
+Welcome to Terrain!  I've mastered the stairs!
```

Comment:
Sadly this message doesn't make as much sense with the new name. Perhaps we should find something else humorous to say ;)